docs/docs/vulnerability/languages/java.md

```diff
@@ -0,0 +1,51 @@
+# Java
+
+## Features
+Trivy supports three types of Java scanning: `jar/war/par/ear`, `pom.xml` and `*gradle.lockfile` files.
+The following table provides an outline of the features Trivy offers.
+
+
+| Artifact         |  Maven repository [^1]   | Dev dependencies |
+|------------------|:------------------------:|:-----------------|
+| jar/war/par/ear  |            ✓             | Include          |
+| pom.xml          |            ✓             | Exclude          |
+| *gradle.lockfile |            -             | Exclude          |
+
+### Jar
+To find information about your jar[^2] file, Trivy parses `pom.properties` and `MANIFEST.MF` files in your jar[^2] file and takes `required properties`[^3].
+
+If those files don't exist or don't contain enough information - Trivy uses [maven repository REST API](https://central.sonatype.org/search/rest-api-guide) to try to get missing information.
+
+!!! Note
+    Trivy only takes information about package. We don't take list of vulnerability for package from `maven repository`(Information about data sources for java you can see [here](../detection/data-source.md))
```

Comment:
"Trivy only takes information about packages. We don't take a list of vulnerabilities for packages from the `maven repository`(Information about data sources for java you can see"

Reply:
Done